src/main/kotlin/org/misarch/address/graphql/input/CreateAddressInput.kt

```diff
@@ -0,0 +1,21 @@
+package org.misarch.address.graphql.input
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+
+/**
+ * Base class for [CreateUserAddressInput] and [CreateVendorAddressInput]
+ */
+abstract class CreateAddressInput(
+    @property:GraphQLDescription("The first part of the street part of the address to create")
+    val street1: String,
+    @property:GraphQLDescription("The second part of the street part of the address to create")
+    val street2: String,
+    @property:GraphQLDescription("The city part of the address to create")
+    val city: String,
+    @property:GraphQLDescription("The postal code part of the address to create")
+    val postalCode: String,
+    @property:GraphQLDescription("The country part of the address to create")
+    val country: String,
+    @property:GraphQLDescription("The company name part of the address to create")
+    val companyName: String
```

Comment:
Why obligatory?

Reply:
it's like this in our domain model
did not find the time yet to change it after our decision to change it, will do soon

Reply:
fixed with ee7e6a3dbd4ff5e703620bdf6c11b6ff9b443f82